### PR TITLE
Use depth instead of hosts in .toml-files

### PR DESCRIPTION
**Pull request by ineiti**
Adds the `depth`-field to toml-files for more easy simulations

### CODE REVIEW

File: simul/simul.go [hunk 1]
```diff
@@ -216,6 +210,22 @@ func RunTest(rc platform.RunConfig) (monitor.Stats, error) {
...
+// -parameter in the Runconfig
+func CheckHosts(rc platform.RunConfig) {
+	hosts, err := rc.GetInt("hosts")
+	if hosts == 0 || err != nil {
+		depth, err1 := rc.GetInt("depth")
+		bf, err2 := rc.GetInt("bf")
+		if depth == 0 || bf == 0 || err1 != nil || err2 != nil {
+			dbg.Fatal("No Hosts and no Depth or BF given - stopping")
+		}
+		hosts = int((1 - math.Pow(float64(bf), float64(depth+1))) /
```
Comment (nikkolasg) [inline simul/simul.go]: elaborate on this formula please ?

Comment (ineiti) [inline simul/simul.go]: https://en.wikipedia.org/wiki/Geometric_series

Root-node: 1
1st level: bf (branching-factor)
2nd level: bf^2 (each child has bf children)
3rd level: bf^3

So total: sum(level=0..depth)(bf^level)

**Comment by nikkolasg** [comment]
OK for me.

**Comment by Liamsi** [comment]
Doesn't compile

**Comment by ineiti** [comment]
merged with test_localhost and running through

File: simul/simul.go [hunk 2]
```diff
@@ -216,6 +210,28 @@ func RunTest(rc platform.RunConfig) (monitor.Stats, error) {
...
+// -parameter in the Runconfig
+func CheckHosts(rc platform.RunConfig) {
+	hosts, err := rc.GetInt("hosts")
+	if hosts == 0 || err != nil {
+		depth, err1 := rc.GetInt("depth")
+		bf, err2 := rc.GetInt("bf")
+		if depth == 0 || bf == 0 || err1 != nil || err2 != nil {
+			dbg.Fatal("No Hosts and no Depth or BF given - stopping")
+		}
+		// Geometric series
```
Comment (Liamsi) [inline simul/simul.go]: Nitpicking: should be geometric *sum* (series is the infinite 'version')

**Comment by Liamsi** [comment]
Looks good

**Pull request merged**